docs/self-hosting/guides/automated-bootstrapping.mdx

```diff
@@ -0,0 +1,160 @@
+---
+title: "Automated Bootstrapping"
+description: "Learn how to provision and configure Infisical instances programmatically without UI interaction"
+---
+
+Infisical's Automated Bootstrapping feature enables you to provision and configure an Infisical instance without using the UI, allowing for complete automation through static configuration files, API calls, or CLI commands. This is especially valuable for enterprise environments where automated deployment and infrastructure-as-code practices are essential.
+
+## Overview
+
+The Automated Bootstrapping workflow automates the following processes:
+- Creating an admin user account
+- Initializing an organization for the entire instance
+- Establishing an **instance admin machine identity** with full administrative permissions
+- Returning the machine identity credentials for further automation
+
+## Key Concepts
+
+- **Instance Initialization**: Infisical requires configuration variables to be set during launch, after which the bootstrap process can be triggered.
+- **Instance Admin Machine Identity**: The bootstrapping process creates a machine identity with instance-level admin privileges, which can be used to programmatically manage all aspects of the Infisical instance.
+  ![Instance Admin Identity](/images/self-hosting/guides/automated-bootstrapping/identity-instance-admin.png)
+- **Token Auth**: The instance admin machine identity uses [Token Auth](https://infisical.com/docs/documentation/platform/identities/token-auth), providing a JWT token that can be used directly to make authenticated requests to the Infisical API.
+
+## Prerequisites
+
+- An Infisical instance launched with all required configuration variables
+- Access to the Infisical CLI or the ability to make API calls to the instance
+- Network connectivity to the Infisical instance
+
+## Bootstrap Methods
```

Comment:
nit: what do you think about using tabs so that we only show one info at a time https://mintlify.com/docs/content/components/tabs